src/legacy/ui/public/vis/editors/default/advanced_toggle.html

```diff
@@ -1,10 +1,11 @@
 <div class="eui-textRight">
   <a ng-click="advancedToggled = !advancedToggled">
-    <i aria-hidden="true" class="fa fa-caret-down" ng-class="{'fa-caret-down': advancedToggled, 'fa-caret-left': !advancedToggled}"></i>
-    <span
-      i18n-id="common.ui.vis.editors.advancedToggle.advancedLinkLabel"
-      i18n-default-message="Advanced"
+    <icon ng-show="advancedToggled" type="'arrowDown'" size="'s'"></icon>
+    <icon ng-show="!advancedToggled" type="'arrowRight'" size="'s'"></icon>
+    <small><span
```

Comment:
We could move the `i18n-*` attributes directly to the `<small>` and get rid of an additional `<span>` element here.